docs/UserGuide.md

```diff
@@ -131,19 +151,27 @@ Examples:
 * `find alex david` returns `Alex Yeoh`, `David Li`<br>
   ![result for 'find alex david'](images/findAlexDavidResult.png)
 
-### Deleting a person : `delete`
+### Deleting a volunteer : `vdelete` [COMING SOON]
```

Comment:
Likewise, Deleting a volunteer from event

Reply:
yep

Reply:
also i think the command should be vremove. because for events we are alr using edelete so it is consistent if we use vdelete to delete a volunteer from the volunteer list, instead of from an event